### PR TITLE
Fix showing cloud compilation limits

### LINKED ISSUE #1779: IDE always shows cloud limits for anonymous
### Environment

* Scope: desktop IDE, browser IDE
* Version: master and earlier
* OS: any
* Machine: any

### Steps to reproduce

1.  Login as some user
2.  Run simulations or tabtests many times
3.  Look at available compilations at 'Daily quotas'

### Expected behavior

It shows limits for logged in user.

### Actual behavior

It shows limits for IP address (anonymous).

### How to fix

Send `Authorization` header with bearer token if user is logged in.
Related code: https://github.com/xodio/xod/blob/0.29.x/packages/xod-client/src/user/actions.js#L17

### LINKED ISSUE #1780: IDE doesn't update cloud limits
### Environment

* Scope: desktop IDE, browser IDE
* Version: master and earlier
* OS: any
* Machine: any

### Steps to reproduce

1.  Run simulations or tabtests many times
2.  Look at available compilations at 'Daily quotas'

### Expected behavior

It doesn't update cloud limits.

### Actual behavior

It updates cloud limits.

### How to fix

Dispatch `updateCompileLimit(false)` after simulation/tabtest run. After failed too.

May be we need to update limits time based too?

Also, Electron version updates limits on 'Upload to Arduino` popup. I think this is not required.

### PR DESCRIPTION
**Pull request by brusherru**
It closes #1779 and closes #1780

### CODE REVIEW

File: packages/xod-client/src/user/actions.js [hunk 1]
```diff
@@ -51,6 +34,69 @@ export const fetchGrant = () => dispatch =>
...
+const fetchLimits = headers => fetch(getCompileLimitUrl(), { headers });
+
+export const updateCompileLimit = (startup = false) => dispatch => {
+  const basicHeaders = startup ? { 'x-launch': 'true' } : {};
+  return dispatch(requestAuthorized(fetchLimits, basicHeaders))
+    .catch(() => null)
+    .then(limit =>
+      dispatch({
+        type: ActionTypes.UPDATE_COMPILE_LIMIT,
+        payload: limit,
```
Comment (knopki) [inline packages/xod-client/src/user/actions.js]: `Compilations: undefined`?  :thinking:

Comment (knopki) [reply packages/xod-client/src/user/actions.js]: ...or `currently offline`?

Comment (brusherru) [reply packages/xod-client/src/user/actions.js]: The second one :)

Comment (brusherru) [reply packages/xod-client/src/user/actions.js]: Okay, I won't update the limits if Promise rejected with an error. ;)

**Pull request merged**